### DISCUSSION TITLE
About sign deb files

### DISCUSSION THREAD

**Discussion by RohitVerma882** [termux/termux-packages#17513] [Q&A] [answered]
I'm created deb file using ```termux-create-package``` and ```termux-apt-repo```
How to sign deb files for publishing?

**Comment by sylirre** [comment, marked as answer]
If you would read `termux-apt-repo` help info, you would find an option to enable signing.

> -s --sign         sign repo with GPG key

You will need to setup GnuPG keys.

**Reply by RohitVerma882** [reply]
Already solved, thanks
